src/main/java/Event.java

```diff
@@ -0,0 +1,17 @@
+public class Event extends Task {
+    protected String startDate;
+    protected String endDate;
+
+    public Event(String description, String startDate, String endDate) {
+        super(description);
+        startDate = startDate.replace("from", "from:");
+        endDate = endDate.replace("to", "to:");
```

Comment:
`startDate` and `endDate` are also recycled here, perhaps consider my recommendation in the prior comment on `Ava.java` from line 64 to 73?